### ISSUE TITLE
Kubectl get cs/componentstatuses returns Unknown in embedded/UI shell

### ISSUE THREAD

**Issue by superseb** [rancher/rancher#23514]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
Setup custom cluster and execute `kubectl get cs` via embedded kubectl in UI.

**Result:**
```
NAME                 AGE
controller-manager   <unknown>
scheduler            <unknown>
etcd-0               <unknown>
etcd-1               <unknown>
etcd-2               <unknown>
```

Workaround is to use `kubectl get cs -o yaml` (or basically anything else than table)

**Other details that may be helpful:**
Upstream issue: https://github.com/kubernetes/kubernetes/issues/83024

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI): v2.3.1
- Installation option (single install/HA): single

**Comment by baskinsy** [comment]
Can confirm the same after upgrading to rancher v2.3.3. Unfortunately hadn't checked before the upgrade.

**Comment by xzy256** [comment]
v1.16.3.  I get same error. How to fix it?

**Comment by spdfnet** [comment]
Seems to be deprecated:
https://github.com/kubernetes/kubernetes/issues/83943

**Comment by superseb** [comment]
Solved in kube-apiserver v1.17.0, workaround available for older versions

**Comment by zhuchance** [comment]
kubectl get cs -o=go-template='{{printf "NAME\t\t\tHEALTH_STATUS\tMESSAGE\t\n"}}{{range .items}}{{$name := .metadata.name}}{{range .conditions}}{{printf "%-24s%-16s%-20s\n" $name .status .message}}{{end}}{{end}}'
